src/components/CallAssignmentList.tsx

```diff
@@ -0,0 +1,43 @@
+import { FormattedMessage as Msg } from 'react-intl';
+import { Flex, Text, View } from '@adobe/react-spectrum';
+
+import { ZetkinCallAssignment, ZetkinMembership } from '../types/zetkin';
+
+interface CallAssignmentListProps {
+    callAssignments: ZetkinCallAssignment[] | undefined;
+    memberships: ZetkinMembership[] | undefined;
```

Comment:
It feels weird to me that a component built to list call assignments requires a list of organizations (or even weirder, memberships like here).

Have you considered augmenting the `ZetkinCallAssignment` type with an `organization` field instead, like we discussed for events?